iati/using_data/templates/using_data/tools_index_page.html

```diff
@@ -42,6 +42,7 @@ <h3 class="aside-m__heading">{% trans "Support" %}</h3>
 
     <div class="space-section-bottom">
```

Comment:
Would it not be better to add the id to this div instead, so we don't add empty elements in page?

Reply:
We might need to test the behavior between the two. The `<a>` element with no content is following the convention that Fffunction has used on the rest of the pages. I think they've got some custom JS implemented that tends to scroll a bit past content, so it may scroll past the `<div>` if we just add the ID there.

Reply:
Jumps here (as far as it can go) because there isn't a page worth of content at the bottom 
![image](https://user-images.githubusercontent.com/2836840/46083041-7fd48e80-c16e-11e8-8097-ca9d0d392243.png)

I'm going to try adding some filler at the bottom to test the full behavior.

Reply:
FWIW I don’t think it’s really semantically correct to use an `<a>` for this anyway. But in general, agree with @samuele-mattiuzzo. Feels like an anti-pattern to me.

Reply:
Yeah that looks good though, it's exactly where is supposed to be (aligned with that div at the bottom), we should go with this

Reply:
Yep, works fine. 
![image](https://user-images.githubusercontent.com/2836840/46083211-ebb6f700-c16e-11e8-91c1-81510c986dc7.png)